en/docs/learn/lite-user-registration.md

```diff
@@ -0,0 +1,79 @@
+# Lite User Registration
+
+Lite user registration lets users register without defining a password. Autheticatioon and user profile creation is initiated merely using the email address. Follow the steps given below to enable and use lite user registration. 
+
+## Enable the LiteUserRegistration handler 
+
+Add the following configuration in the `<IS_HOME>/repository/conf/deployment.toml` file to enable lite user registration.
+
+```toml
+[identity_mgt.events.schemes.liteUserRegistration.properties]
+enable = true
+```
+
+## Enable Lite User Registration 
+
+1.	Sign in to the WSO2 Identity Server [Management Console](../../setup/getting-started-with-the-management-console).
+
+2.	On the **Main** menu, click **Identity > Identity Providers > Resident**. 
+
+3.	Expand **User Onboarding**. 
+
+4.	Expand **Lite User Registration**. 
+
+5.	Select **Lite user registration** to allow users to register without a password. 
+
+	![resident-idp](../assets/img/learn/resident-idp.png)
+
+6. Click **Update**. 
+
+## Map local claim attribute 
+
+1.	On the **Main** menu, click **Identity > Claims > List**. 
+
+2.	Select `http://wso2.org/claims`. 
+
+3.	From the list of available claims, select **Lite User** and click **Edit**. 
+
+4.	Change the value of **Mapped Attribute** to `title`. 
+
+	![mapped-attribute-change](../assets/img/learn/mapped-attribute-change.png)
+
+	!!! note 
+		This step needs to be performed since the `isLiteUser` attribute has not been introduced yet in the product. 
+
+
+## Create a Lite User
+
+Execute the following cURL command to create a user using the email address alone. 
+
+!!! note 
+	Make sure the user store regex patterns allow email to be used as the user name
+
+```curl 
+curl -X POST -H "Authorization: Basic YWRtaW5Ad3NvMi5jb206YWRtaW4=" -H "Content-Type: application/json" -d '{"email
+": "<regsiter_email>","realm": "PRIMARY","preferedChannel":"Email","claims":[], "properties": []}' "https://localhost:9443/api/identity/user/v1.0/lite"
+```
+
+This will also trigger an email with a confirmation code. 
+
+## Introspect the code 
+
+Execute the following command to validate the confirmation code. 
+
+```curl 
+curl -k -v -X POST -H "Authorization: Basic YWRtaW5Ad3NvMi5jb206YWRtaW4=" -H "Content-Type: application/json" -d '{ "code": "17f00958-a1d7-47b9-8183-be99c08a800f"}' "https://dev.portal.asgardio.io/api/identity/user/v1.0/introspect-code"
+```
+
+This command validates if the provided confirmation code is valid or not along with other details. Based on this response, the user can make changes to the application as per the requirement and use this confirmation code as the authorization input.  
```

Comment:
Shall we reword this as "Further scenario executions can be implemented to consume this valid confirmation and let the user build us the user profile providing a password and create tenants etc. With lite user registration, minimum resources are consumed until a user proves to have a valid email address."